shell/browser/electron_browser_client.cc

```diff
@@ -297,6 +297,12 @@ breakpad::CrashHandlerHostLinux* CreateCrashHandlerHost(
 }
 
 int GetCrashSignalFD(const base::CommandLine& command_line) {
+  if (crash_reporter::IsCrashpadEnabled()) {
+    int fd;
+    pid_t pid;
+    return crash_reporter::GetHandlerSocket(&fd, &pid) ? fd : -1;
```

Comment:
_micronit_ we can pass a nullptr pid here

Reply:
True, though this is copy/pasted from Chrome: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/chrome_content_browser_client.cc;l=968;drc=161239673fd3fac73ecbf3d4eedb770f3e0d7fa2